dockernet/config/relayer_config_ics.yaml

```diff
@@ -16,6 +16,7 @@ chains:
       gas-prices: 0.02ustrd
       debug: false
       timeout: 20s
+      min-loop-duration: 50ms
```

Comment:
these loops are pretty aggressive, i'd recommend removing it or setting it to 1s

Reply:
468288391

Reply:
I just tagged this one instance, but applies to all the ones in the other config files too

Reply:
d4783a8db